utils/cups-browsed.c

```diff
@@ -8462,13 +8464,15 @@ gboolean update_cups_queues(gpointer unused) {
 	  } else if (strncmp(line, "*End", 4)) {
 	    /* Write an "APRemoteQueueID" line to make this queue marked
 	       as remote printer by CUPS */
-	    if (p->netprinter == 0 &&
-		strncmp(line, "*%", 2) &&
-		strncmp(line, "*PPD-Adobe:", 11) &&
-		ap_remote_queue_id_line_inserted == 0) {
-	      ap_remote_queue_id_line_inserted = 1;
-	      cupsFilePrintf(out, "*APRemoteQueueID: \"\"\n");
-	    }
+    if (!AllowResharingRemoteCUPSPrinters) {
+        if (p->netprinter == 0 &&
+            strncmp(line, "*%", 2) &&
+            strncmp(line, "*PPD-Adobe:", 11) &&
+            ap_remote_queue_id_line_inserted == 0) {
+          ap_remote_queue_id_line_inserted = 1;
+          cupsFilePrintf(out, "*APRemoteQueueID: \"\"\n");
+        }
+      }
```

Comment:
Could you adjust the indentation of this part, so that it matches the surrounding code? All the rest seems OK.

Reply:
@tillkamppeter, Does it look good now?

Reply:
As the existing code uses tabs which represent 8 spaces you need to have your editor display the code with the tabs representing 8 spaces and then adjust your code to the existing one. For me it looks like that your editor is showing tabs as only 2 or 4 spaces.

Reply:
> For me it looks like that your editor is showing tabs as only 2 or 4 spaces.

You are right. I've changed my editor's configuration now. Could you review it now please?

Reply:
Now it is all OK, thanks.

Reply:
Thanks @tillkamppeter.